network_visualizer/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>network_visualizer</name>
+  <version>0.0.0</version>
+  <description>GUI for visualizing UW React path planning Neural Network</description>
+
+  <maintainer email="dev@uwreact.ca">UW React Software Team</maintainer>
+
+  <license>BSD-3</license>
+
+  <url type="website">http://uwreact.ca</url> 
+  <url type="repository">http://github.com/uw_react/uwreact_robot/</url> 
+  <url type="bugtracker">http://github.com/uw_react/uwreact_robot/issues</url> 
+  
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>python_qt_binding</build_depend>
```

Comment:
Prefer a single `<depend>` rather than each `<build_depend>`, `<build_export_depend>`, and `<exec_depend>`

Reply:
Should be a `depend`, not `build_depend`